docs/tutorial/planning.txt

```diff
@@ -0,0 +1,82 @@
+My current plan for my COMBINE 25 tutorial (that will then be used for the spras doc tutorials)
+- I will be testing this on a user that has basic python knowledge but doesn't know spras
+
+*Tony will be giving a presentation on PRAs and SPRAS prior (hopefully) to this tutorial
+
+1) basic
+Goal: Get new users comfortable installing SPRAS
+- don't need to preinstall conda, docker, vscode(?) (or run on terminal), and cytoscape prior, but it is recommended to to make it easier/faster
```

Comment:
We should set installation expectations. If we want participants to follow along, they should have a few minimum dependencies like Docker and conda installed already.